crates/solver/src/solver.rs

```diff
@@ -377,6 +379,12 @@ pub fn create(
                 "configured slippage",
             );
 
+            let score_calculator = score_configuration.get_calculator(solver_type);
+            tracing::debug!(
+                solver = ?solver_type, score = ?score_calculator,
+                "configured score",
+            );
```

Comment:
This information already gets printed on startup as part of the CLI arguments.

Reply:
removed